.gitattributes

```diff
@@ -0,0 +1,11 @@
+* text=auto
```

Comment:
This is usually a good opportunity to enforce a single line ending regardless of the OS with `* text eol=lf`. This solves _many_ issues with bizarre diffs in cross-platform development.